Adds react-dropzone-uploader subdomain

kylebebak opened the pull request:
I'd like <https://fortana-co.github.io/react-dropzone-uploader> to point to <https://react-dropzone-uploader.js.org>.

I've added a CNAME file here to prove I own the repo.

I added it to `master` instead of `gh-pages`, because if I add it to the `gh-pages` branch GitHub redirects all petitions from <https://fortana-co.github.io/react-dropzone-uploader> to <https://react-dropzone-uploader.js.org>, which can't respond to requests yet.


- [X] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [X] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to move the```CNAME``` to the correct branch to make it work (make sure to first remove the CNAME before you add the new one. Otherwise the new one won't get evaluated).
I'll merge after we have "contact".

kylebebak commented:
Awesome, I just removed the CNAME from `master` and added it to `gh-pages`. My site was up at <https://react-dropzone-uploader.js.org> in less than a minute!

Thanks a lot, and thanks for building js.org.


It might be a good idea to allow people to create the CNAME file in `master`, instead of in `gh-pages`, to prove they own their docs site/repository.

The reason I say this is because GitHub begins redirecting petitions to `org.github.io/project` to `project.js.org` as soon as the CNAME file is created in the `gh-pages` branch.

`project.js.org` doesn't respond to petitions until you add the subdomain to **js.org**'s zonefile, which means the docs site will be down during the evaluation period. I was also worried that you wouldn't be able to see the site's content, and that it would be rejected based on the No Content policy.

The No Content policy strikes me as a very good idea by the way!


In any case, thanks again!

kylebebak commented:
@indus 

Ah, thanks for that explanation!
